otter/log/spec.py

```diff
@@ -0,0 +1,91 @@
+"""
+Format logs based on specification
+"""
+
+from twisted.python.failure import Failure
+
+
+# mapping from msg type -> message
+msg_types = {
+    "execute-convergence": "Executing convergence",
+    "execute-convergence-results": (
+        "Got result of {worst_status} after executing convergence"),
+    "launch-servers": "Launching {num_servers} servers",
+    "delete-server": "Deleting {server_id} server",
+    "add-server-clb": ("Adding {server_id} with IP address {ip_address} "
```

Comment:
Non-blocking - we're using the bulk add/bulk delete apis for CLB, so wondering if these are going to end up being `server_ids` and `ip_addresses`.  But that can be fixed later when we do the actual logging.